wild/tests/sources/exclude-section.s

```diff
@@ -0,0 +1,11 @@
+.section .keep, "aSM", @progbits, 1
```

Comment:
At the moment both sections are string-merge sections ("S" and "M" flags). I'd probably keep this file somewhat minimal. Perhaps even get rid of the symbols so that the only difference is the exclude bit (at the moment one of the sections has a symbol that is referenced, which prevents GC if gc-sections is enabled). Something like this:

```
.section .keep
.ascii "ShouldBeKept\0"

.section .exclude, "e"
.ascii "ShouldBeExcluded\0"
```